private/rules/maven_utils.bzl

```diff
@@ -85,13 +92,20 @@ def generate_pom(
     deps = []
     for dep in sorted(versioned_dep_coordinates) + sorted(unversioned_dep_coordinates):
         include_version = dep in versioned_dep_coordinates
-        unpacked = _unpack_coordinates(dep)
+        split = dep.split(":")
+        if len(split) == 5 and split[3] not in ["import", "compile", "runtime", "test", "provided", "system"]:
```

Comment:
This seems hacky.

Reply:
I had to convert it to the gradle format for it to be parsed correctly. If I just pass it in as-is, then `unpack_coordinates` is putting the `classifier` into the `<scope>` tag.
So maybe this hardcoded check for `"import", "compile", "runtime", "test", "provided", "system"` should go into the `unpack_coordinates`?

I feel like I might be missing something obvious if you are saying the heuristic check is already in place.